Assets/Mirror/Runtime/Obsolete.cs

```diff
@@ -0,0 +1,384 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using UnityEngine;
+
+/// <summary>
+/// This file contains all classes with deprecated methods and properties.
+/// We created this as a convenient reference, and to make it easier for users
+/// migrating from UNet to Mirror.  This is the only file that is packed with
+/// multiple classes, otherwise we follow the practice of one class per file in
+/// the project.
+/// 
+/// In many cases the methods here are written to directly call their replacements
+/// so you likley can copy that code to your own project to clear the obsolete warnings.
+/// 
+/// Those wishing to submit a PR that would obsolete something in a core class must move
+/// the obsoleted property or method to this file with appropriate comments and include
+/// the changes here with the PR.
```

Comment:
This text should probably be added to contrib guidelines @vis2k

Reply:
Separate PR